node/src/main/java/eu/cloudnetservice/node/provider/NodeGroupConfigurationProvider.java

```diff
@@ -191,15 +192,24 @@ protected void writeAllGroupConfigurations() {
 
   protected void loadGroupConfigurations() {
     FileUtil.walkFileTree(GROUP_DIRECTORY_PATH, ($, file) -> {
+      var document = JsonDocument.newDocument(file);
+
+      // TODO: remove in 4.1
+      // check if the task has environment variables
```

Comment:
We are talking about groups here

Reply:
Not really relevant, we're removing these lines in a few versions anyway